package-lock.json

```diff
@@ -10121,9 +10121,9 @@
       }
     },
     "lodash": {
-      "version": "4.17.15",
-      "resolved": "https://registry.npmjs.org/lodash/-/lodash-4.17.15.tgz",
-      "integrity": "sha512-8xOcRHvCjnocdS5cpwXQXVzmmh5e5+saE2QGoeQmbKmRS6J3VQppPOIt0MnmE+4xlZoumy0GPG0D0MVIQbNA1A=="
+      "version": "4.17.20",
```

Comment:
We try to keep each PR addressing 1 issue (and only that issue). While these updates are good (and will probably be done sometime in the future), it isn't needed to solve this issue.

Reply:
Understand that. Instead of changing css of .ngb-tooltip-class, maybe I should just add space above the element in HTML page? This way should not affect other pages.

Reply:
We would prefer not to hardcode styles in this manner, as this styling may be hard to trace down and doesn't scale well. One thing you could try is adding a class to all such "Evaluee/Recipient" text that would be generated and have that class be positioned slightly lower. That way it doesn't affect other items that are not using that class.

Reply:
copy that

> We would prefer not to hardcode styles in this manner, as this styling may be hard to trace down and doesn't scale well. One thing you could try is adding a class to all such "Evaluee/Recipient" text that would be generated and have that class be positioned slightly lower. That way it doesn't affect other items that are not using that class.